libs/auth/src/dipped_smart_card_auth.ts

```diff
@@ -180,7 +183,19 @@ export class DippedSmartCardAuth implements DippedSmartCardAuthApi {
     input: { pin: string }
   ): Promise<void> {
     await this.checkCardReaderAndUpdateAuthStatus(machineState);
-    const checkPinResponse = await this.card.checkPin(input.pin);
+    let checkPinResponse: CheckPinResponseExtended;
+    try {
+      checkPinResponse = await this.card.checkPin(input.pin);
```

Comment:
Rather than having card API implementations remember to convert errors to `CheckPinResponseError` objects, I decided it was safer to just have the auth classes take care of this